### ISSUE TITLE
props maybe not immutable

### ISSUE THREAD

**Issue by georgebbbb** [jurassix/react-immutable-render-mixin#10]
examples
```javascript
<My value={Immutable.fromJS({value:123}) }  onChange={this.handleChange.bind(this)}></My>
```
this component `s props isn`t immutable 
so 
```javascript
is(objA, objB)
-> false
```

**Comment by jurassix** [comment]
@georgebbbb technically onChange is a property of your Component and should really be something similar to this:

```js
const props = {
  value: 123,
  onChange: this.handleChange.bind(this),
}
<My {...Immutable.fromJS(props)} />
```

**Comment by jurassix** [comment]
I'm going to leave this open to see if anyone has other solutions.

**Comment by georgebbbb** [comment]
Thank you
but It seems that ` {...Immutable.fromJS(props)}` does not work 
https://github.com/facebook/react/issues/2059
[GitHub](https://github.com/facebook/react/issues/2059)

**Comment by jurassix** [comment]
Yeah I can see that my example above doesn't make sense. I'll take a look this weekend and add a test case to cover this issue. Feel free to do the same. I think there is an obvious solution, I just need a test to flush it out.

**Comment by georgebbbb** [comment]
ok,thank you very much

**Comment by jurassix** [comment]
@georgebbbb I still haven't added any tests for this issue. But, there may be a quick fix for you.

Instead of call _bind()_ every render, try calling bind once, in the constructor of your Component. This should keep the __onChange()__ handler referentially equal in child components. 

```js
class App extends React.Component {
  constructor() {
    super();
    this.handleChange = this.handleChange.bind(this);
  }
  render() {
    return (
      <My value={Immutable.fromJS({value:123}) }  onChange={this.handleChange}></My>
    );
  }
}
```

Let me know if this fixes the issue.

**Comment by jurassix** [comment]
@georgebbbb were you able to confirm the above solved the issue?
